internal/testrunner/runners/system/runner.go

```diff
@@ -452,7 +452,8 @@ func (r *runner) tearDownTest(ctx context.Context) error {
 	}
 	_, err := r.resourcesManager.ApplyCtx(cleanupCtx, r.resources(resourcesOptions))
 	if err != nil {
-		return err
+		logger.Errorf("Failed to uninstall package is there any other execution in parallel?: %s", err)
+		// return err
```

Comment:
Should it be reported as a Warn level ?

Reply:
> It would be interesting to install package just once before running all tests. And once all tests have finished, uninstall that package.

Related to this, a new PR has been created: https://github.com/elastic/elastic-package/pull/1845